src/liballoc/tests/binary_heap.rs

```diff
@@ -282,7 +282,7 @@ fn assert_covariance() {
 //
 // Destructors must be called exactly once per element.
 #[test]
-#[cfg(not(miri))]
+#[cfg(not(miri))] // Miri does not support panics
```

Comment:
I'm surprised that miri doesn't given that you can `panic!(..)` inside `const` items...

Reply:
Well, the more precise statement is that Miri aborts execution on a panic. Which is not helpful for these tests.

Reply:
Oh, okay. That makes sense. Thanks for clarifying :)